crates/lib/src/client.rs

```diff
@@ -22,6 +22,9 @@ pub use qcs_api_client_common::configuration::LoadError;
 pub use qcs_api_client_grpc::channel::Error as GrpcError;
 pub use qcs_api_client_openapi::apis::Error as OpenApiError;
 
+const DEFAULT_MAX_MESSAGE_ENCODING_SIZE: usize = 50 * 1024 * 1024;
+const DEFAULT_MAX_MESSAGE_DECODING_SIZE: usize = 50 * 1024 * 1024;
```

Comment:
We have a similar constant in [the qpu api](https://github.com/rigetti/qcs-sdk-rust/blob/main/crates/lib/src/qpu/api.rs#L42). Should we try to consolidate around one?

Reply:
I did see that, but opted for a module-specific one simply because the conserv-related constant is set to 250MB which seemed unnecessarily high for translation messages.